Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -577,6 +578,22 @@ private IEnumerator<BaseData> GetNextTicksEnumerator()
             Log.Trace("LiveTradingDataFeed.GetNextTicksEnumerator(): Exiting enumerator thread...");
         }
 
+        /// <summary>
+        /// Will wrap the provided enumerator with a <see cref="FrontierAwareEnumerator"/>
+        /// using a <see cref="StepTimeProvider"/> that will advance time based on the provided
+        /// function
+        /// </summary>
+        /// <remarks>Won't advance time if now: bigger than 23pm, less than 6am or Saturday</remarks>
+        private IEnumerator<BaseData> GetConfiguredFrontierAwareEnumerator(IEnumerator<BaseData> enumerator,
+            TimeZoneOffsetProvider tzOffsetProvider)
+        {
+            var stepTimeProvider = new StepTimeProvider(_frontierTimeProvider,
+                // advance time if before 23pm or after 5am and not on Saturdays
+                time => time.Hour < 23 && time.Hour > 5 && time.DayOfWeek != DayOfWeek.Saturday);
```

Comment:
These times/days of weeks seem very much coupled to particular security types. The documentation states _what_ it does but not _why_ it does it.

Reply:
Good point! I'll add _why_ it does it